lib/hooks/useTimesheetEntries.ts

```diff
@@ -51,6 +51,24 @@ export const useEmployeeTimesheetEntries = (
     return { isSuccess: false, isError: false, isLoading: true }
 }
 
+export const useTimeIntervalTimesheetEntries = (
+    user: User
+): ApiGetResponse<TimesheetEntry[]> => {
+    const startDate = "2022-01-01"
+    const endDate = "2022-12-12"
+    if (startDate && endDate !== null) {
```

Comment:
This conditional statement is redundant since `startDate` and `endDate` are always defined.